tests/netebpfext_unit/netebpf_ext_helper.h

```diff
@@ -14,8 +14,9 @@
 #endif
 
 #include "..\..\external\usersim\src\fwp_um.h"
+#include "..\libs\store_helper\kernel\ebpf_registry_helper_km.h"
```

Comment:
I think we should remove line 17.   This file need not know there is any "registry" dependency.
Also it is odd to include both a "um" file (line 16) and a "km" file (line 17) without explanation.

Reply:
cleaned-up